pkg/proxy/iptables/proxier.go

```diff
@@ -157,6 +155,30 @@ type serviceInfo struct {
 	serviceLBChainName       utiliptables.Chain
 }
 
+// ClusterIP is part of interface.
+func (info *serviceInfo) ClusterIP() string {
+	return info.clusterIP.String()
+}
+
+// Port is part of interface.
```

Comment:
Can you change the comment to something like "Port is defined in the ServicePort interface"? This makes it clear what interface you are referring to.

Reply:
What about "Port is part of ServicePort interface"?

Reply:
Done